quick-protobuf/examples/pb_rs_v3/data_types.rs

```diff
@@ -168,75 +210,78 @@ impl<'a> MessageRead<'a> for FooMessage<'a> {
 impl<'a> MessageWrite for FooMessage<'a> {
     fn get_size(&self) -> usize {
         0
-        + if self.f_int32 == 0i32 { 0 } else { 1 + sizeof_varint(*(&self.f_int32) as u64) }
-        + if self.f_int64 == 0i64 { 0 } else { 1 + sizeof_varint(*(&self.f_int64) as u64) }
-        + if self.f_uint32 == 0u32 { 0 } else { 1 + sizeof_varint(*(&self.f_uint32) as u64) }
-        + if self.f_uint64 == 0u64 { 0 } else { 1 + sizeof_varint(*(&self.f_uint64) as u64) }
-        + if self.f_sint32 == 0i32 { 0 } else { 1 + sizeof_sint32(*(&self.f_sint32)) }
-        + if self.f_sint64 == 4i64 { 0 } else { 1 + sizeof_sint64(*(&self.f_sint64)) }
-        + if self.f_bool == true { 0 } else { 1 + sizeof_varint(*(&self.f_bool) as u64) }
-        + if self.f_FooEnum == data_types::FooEnum::FIRST_VALUE { 0 } else { 1 + sizeof_varint(*(&self.f_FooEnum) as u64) }
-        + if self.f_fixed64 == 0u64 { 0 } else { 1 + 8 }
-        + if self.f_sfixed64 == 0i64 { 0 } else { 1 + 8 }
-        + if self.f_fixed32 == 0u32 { 0 } else { 1 + 4 }
-        + if self.f_sfixed32 == 0i32 { 0 } else { 1 + 4 }
-        + if self.f_double == 0f64 { 0 } else { 1 + 8 }
-        + if self.f_float == 0f32 { 0 } else { 1 + 4 }
-        + if self.f_bytes == Cow::Borrowed(b"") { 0 } else { 1 + sizeof_len((&self.f_bytes).len()) }
-        + if self.f_string == "" { 0 } else { 2 + sizeof_len((&self.f_string).len()) }
-        + self.f_self_message.as_ref().map_or(0, |m| 2 + sizeof_len((m).get_size()))
-        + self.f_bar_message.as_ref().map_or(0, |m| 2 + sizeof_len((m).get_size()))
-        + if self.f_repeated_int32.is_empty() { 0 } else { 2 + sizeof_len(self.f_repeated_int32.iter().map(|s| sizeof_varint(*(s) as u64)).sum::<usize>()) }
-        + if self.f_repeated_packed_int32.is_empty() { 0 } else { 2 + sizeof_len(self.f_repeated_packed_int32.iter().map(|s| sizeof_varint(*(s) as u64)).sum::<usize>()) }
+        + if self.f_int32 != 0i32 { 1 + sizeof_varint((self.f_int32) as u64) } else { 0 }
```

Comment:
Same here, this change in the generated code seems unnecessary :)

Reply:
Whoop, yeah I think this was just changed by accident from the original as I rewrote some parts of the function; I'll change it back